editor/store/persist.js

```diff
@@ -44,7 +44,7 @@ export function loadAndPersist( store, reducer, reducerKey, storageKey ) {
 	const persistedString = window.localStorage.getItem( storageKey );
 	if ( persistedString ) {
 		const persistedState = {
-			...get( reducer( undefined, { type: 'DEFAULTS' } ), reducerKey ),
+			...get( reducer( undefined, { type: '@@INIT' } ), reducerKey ),
```

Comment:
Redux's initialization action type is intentionally obscure so as not to be targeted directly, so it may be unadvisable for us to attempt to mimic it exactly, vs. something equally obscure but not tied in any way to Redux (our own namespace, for example, e.g. `@@gutenberg/init`), or even an `undefined` type, though this could be error-prone if a reducer or middleware expected `type` to exist as a string.